stl/inc/__msvc_all_public_headers.hpp

```diff
@@ -37,18 +37,52 @@
 #define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING
 #define _SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS
 
+// Core STL Headers
+#include <bit>
+#include <compare>
+#include <concepts>
+#include <coroutine>
+#include <initializer_list>
+#include <limits>
+#include <numbers>
+#include <ratio>
+#include <type_traits>
+#include <utility>
+#include <version>
+
+// Core C Wrapper Headers
+#include <cassert>
+#include <cctype>
+#include <cerrno>
+#include <cfenv>
+#include <cfloat>
+#include <cinttypes>
+#include <climits>
+#include <clocale>
+#include <csetjmp>
+#include <csignal>
+#include <cstdarg>
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <ctime>
+#include <cuchar>
+#include <cwchar>
+#include <cwctype>
+
+#ifndef _EXCLUDE_NON_CORE_HEADERS

```

Comment:
There are three negatives on this line; consider changing `_EXCLUDE_NON_CORE_HEADERS` to `_CORE_HEADERS_ONLY`.

Reply:
Excellent point; changed.